test/Microsoft.AspNetCore.Blazor.Build.Test/ChildContentRazorIntegrationTest.cs

```diff
@@ -181,6 +211,39 @@ public void Render_AttributeChildContent_IgnoresWhitespaceBody()
                 frame => AssertFrame.Text(frame, "hi", 1));
         }
 
+        [Fact]
+        public void Render_MultipleChildContent()
+        {
+            // Arrange
+            AdditionalSyntaxTrees.Add(RenderMultipleChildContent);
+
+            var component = CompileToComponent(@"
+@addTagHelper *, TestAssembly
+@{ RenderFragment<string> header = @<div>@context.ToLowerInvariant()</div>; }
```

Comment:
We're changing this syntax later, right? (to `value => @<div>@value.ToLowerInvariant()</div>`)

Reply:
Yes, this was changed in a later commit.